tests/test_views.py

```diff
@@ -1,15 +1,22 @@
 import pytest
+from urllib.parse import urlencode
+import usaddress
 
 
 def test_api_parse_succeds(client):
-    # TODO: Finish this test. Send a request to the API and confirm that the
-    # data comes back in the appropriate format.
     address_string = '123 main st chicago il'
-    pytest.fail()
+    test_address_components, test_address_type = usaddress.tag(address_string)
+    get_param = urlencode({'address': address_string}) 
 
+    response = client.get(f'/api/parse/?{get_param}')
+    assert response.status_code == 200
+    assert response.data['input_string'] == '123 main st chicago il' 
```

Comment:
What do you think of using the `address_string` here, instead of repeating the full string?
```suggestion
    assert response.data['input_string'] == address_string
```

Reply:
Yes - I think using the previously defined variable as the testing condition vs. using the full string twice is a much better application of the DRY principle and reduces potential error! So I have gone ahead and updated for this change.